k8s/client.go

```diff
@@ -187,8 +187,6 @@ func (c *client) createIngressSource() {
 }
 
 func (c *client) GetServices() ([]*v1.Service, error) {
-	c.createServiceSource()
```

Comment:
I can't think of any reasons why this was needed, given that similarly to ingresses we first watch services before we attempt to retrieve them.